src/components/error/FrontEnd/FrontEndError.react.js

```diff
@@ -40,9 +41,12 @@ class FrontEndError extends Component {
                     </span>
                 </span>
                 <span className="dash-fe-error-top__group">
-                    <span className="dash-fe-error__timestamp">
+                    <span className="dash-fe-error__timestamp percy-hide">
                         {`${e.timestamp.toLocaleTimeString()}`}
                     </span>
+                    <span className="dash-fe-error__timestamp percy-show">
+                        {'12:49:14 PM'}
```

Comment:
:cow2:  maybe a special hardcode time like 00:00:00 to remind this is a special hack of percy?

Reply:
good idea. I'll keep the `PM` so that the width is the same